packages/web-app-user-management/src/views/Users.vue

```diff
@@ -177,7 +177,6 @@ export default defineComponent({
 
       users.value.forEach((user) => {
         user.memberOf = user.memberOf || []
-        user.passwordProfile = user.passwordProfile || { password: '' }
```

Comment:
I don't want to have it in the user object itself anymore, we don't retrieve those data on a get request and just send it while updating the user